pkp/pkp-lib
# pkp/pkp-lib#9381 Limit reviewer statistics to the current journal

### How to test?

There are many ways to test this configuration. The test below illustrates the configuration using the “Active” statistics which indicates if a reviewer is already reviewing another article.  

- Use an OJS instance with at least 2 journals on it.
- Ensure that at least one reviewer is associated with both journals.
- Make a new submission in journal A and send it to review stage.
- Invite the reviewer.
- Connect as the reviewer and accept the invitation.
- Make a new submission in journal B and send it to review stage.
- Invite the reviewer and in the invite window:
  - If show_local_reviewer_statistics configuration is set to “On”, in the reviewer selection window, the reviewer will not appear as having 1 active review.
  - If show_local_reviewer_statistics configuration is set to "Off", in the reviewer selection window, the reviewer will appear as having 1 active review.

Use the configuration below in the **[interface]** section.

```
; Display local reviewer statistics (restricted to the current journal)
show_local_reviewer_statistics = On
```
